instrumentation/opentelemetry-instrumentation-django/tests/test_middleware.py

```diff
@@ -806,7 +810,7 @@ def test_wsgi_metrics_both_semconv(self):
         _recommended_attrs = {
             "http.server.active_requests": active_count_both_attrs,
             "http.server.request.duration": _server_duration_attrs_new,
-            "http.server.duration": _server_duration_attrs_old,
+            "http.server.duration": _django_old_server_duration_attrs,
```

Comment:
I really think this should be more concrete to prevent other similar mistakes slipping through, with something like this:

https://github.com/open-telemetry/opentelemetry-python-contrib/blob/c45a6201e234f73322fcbcb858e906a55d349796/instrumentation/opentelemetry-instrumentation-fastapi/tests/test_fastapi_instrumentation.py#L577-L602

Reply:
I agree. We probably would want to rewrite all instrumentation tests to better reflect the attributes actually expected instead of simply comparing with a static list. At least for this PR, it would be better to fix this regression separately from the larger effort of rewriting these tests and to have a separate PR to address that.

Reply:
It's genuinely hard right now to tell if this PR (or past versions of it) are doing the correct thing.

Reply:
I've added some stricter tests to assert the actual values of expected attributes. Please take a look if they fall along the lines of what you'd expect.